pkger/clone_resource.go

```diff
@@ -87,6 +87,38 @@ func convertCellView(cv cellView) chart {
 	case influxdb.SingleStatViewProperties:
 		setCommon(chartKindSingleStat, p.ViewColors, p.DecimalPlaces, p.Queries)
 		setNoteFixes(p.Note, p.ShowNoteWhenEmpty, p.Prefix, p.Suffix)
+	case influxdb.ScatterViewProperties:
+		ch.Kind = chartKindScatter
+		ch.Queries = convertQueries(p.Queries)
+		ch.Colors = v1ColorsToViewColors(p.ViewColors)
+		ch.XCol = p.XColumn
+		ch.YCol = p.YColumn
+		ch.XDomain = p.XDomain
+		ch.YDomain = p.YDomain
+
+		xAxis := axis{Label: p.XAxisLabel, Prefix: p.XPrefix, Suffix: p.XSuffix, Name: "x"}
+		yAxis := axis{Label: p.YAxisLabel, Prefix: p.YPrefix, Suffix: p.YSuffix, Name: "y"}
+
+		ch.Axes = []axis{xAxis, yAxis}
+		ch.Note = p.Note
+		ch.NoteOnEmpty = p.ShowNoteWhenEmpty
+	case influxdb.HeatmapViewProperties:
+		ch.Kind = chartKindHeatMap
+		ch.Queries = convertQueries(p.Queries)
+		ch.Colors = v1ColorsToViewColors(p.ViewColors)
+		ch.XCol = p.XColumn
+		ch.YCol = p.YColumn
+		ch.XDomain = p.XDomain
```

Comment:
this feels better off in the axes and not the root level of the chart. What's the motivation here?